.github/workflows/npm-publish.yml

```diff
@@ -18,7 +18,12 @@ jobs:
         with:
           node-version: '20.x'
           registry-url: 'https://registry.npmjs.org'
-      - run: npm ci
-      - run: npm publish --access public
+          scope: '@newfold'
+
+      - name: Install Dependencies
+        run: npm i && npm run build
```

Comment:
@circlecube  We should probably use `npm ci` here. It runs quicker and will never result in an update to the lock file, unlike the install command.